src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/DirectionHandler.java

```diff
@@ -50,6 +50,7 @@ public boolean canHandle(
             callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
             && Arrays.asList(
                 Event.SEND_DIRECTION,
+                Event.CHANGE_DIRECTION_DUE_DATE,
```

Comment:
We can't handle `Event.CHANGE_DIRECTION_DUE_DATE` within this handler because new Direction is created which we do not want.

Functional test failed "RIA-602/RIA-723 Change direction due date - submit editable directions"